### PR TITLE
fix: Add documentation for interface support

### PR DESCRIPTION
**Pull request by wtrocki**
Every change we do - skipping some trivial ones will need to land into release notes.
There is no need for documentation for this feature.

### CODE REVIEW

File: docs/releases.md [hunk 1]
```diff
@@ -8,6 +8,33 @@ title: Releases
 This file contains changes and migration steps for Graphback project. 
 Please follow indidividual releases for more information.
 
+# Next
+
+### Graphback
+#### Features
```
Comment (MStokluska) [inline docs/releases.md]: Correct if I'm wrong but don't we need to keep the same format as others? ie 
```
## 0.8.3 (29th August, 2019)
### Templates
#### Fixes
```

File: docs/releases.md [hunk 2]
```diff
@@ -8,6 +8,33 @@ title: Releases
...
+
+- Support for Interfaces in GraphQL schema
+GraphQL Schema supports now interfaces that can be used to ensure consistency of the data.
```
Comment (craicoverflow) [inline docs/releases.md]:
```suggestion
GraphQL Schema now supports interfaces that can be used to ensure consistency of the data.
```

File: docs/releases.md (hunk 1 above)
Comment (craicoverflow) [reply docs/releases.md]: Yes, I am also unsure about this. This new section doesn't really seem to "fit" in with the rest.

Comment (wtrocki) [reply docs/releases.md]: It is not clear what will be the next version so I'm using Next to stage it and going to replace it on release.

Format is right see bellow:

```
## 0.8.0 (28th August, 2019)
### Graphback
#### Features	
```

Comment (wtrocki) [reply docs/releases.md]: > ## 0.9.0 (5th September, 2019)

So I do not know the version number and date- this is going to be done on release.

**Pull request merged**